package.json

```diff
@@ -51,21 +49,20 @@
     "@storybook/react": "^4.0.0-alpha.1",
     "@storybook/storybook-deployer": "^2.3.0",
     "babel-cli": "6.26.0",
-    "babel-core": "6.24.0",
+    "babel-core": "^6.26.0",
     "babel-preset-env": "1.6.1",
     "babel-preset-react": "6.24.1",
     "codecov": "^3.0.0",
     "copyfiles": "^2.0.0",
     "enzyme": "^3.3.0",
     "enzyme-adapter-react-16": "^1.1.1",
     "eslint": "^4.19.1",
-    "eslint-config-airbnb-base": "^12.1.0",
     "eslint-config-edx": "^4.0.3",
-    "eslint-plugin-import": "^2.10.0",
-    "extract-text-webpack-plugin": "^3.0.2",
+    "extract-text-webpack-plugin": "^4.0.0-beta.0",
```

Comment:
Should we be using a beta npm package in production?

Reply:
Good question - it's not ideal but it's the `extract-text-webpack-plugin` version associated with `webpack@4`.

Also, we're just using `webpack` and `extract-text-webpack-plugin` to render the Storybook page so I'm less worried about it - if we were using it to build the Banner Component directly I'd be more concerned.